### ISSUE TITLE
[VSCode] relay-compiler binary not correctly detected in project with PnP

### ISSUE THREAD

**Issue by tobias-tengler** [facebook/relay#4164]
The VSCode extensions assumes the compiler binary to be below `node_modules/relay-compiler`. This assumptions doesn't hold true of course, if the project is using PnP with Yarn 2+ for example.

Should (or rather can) the extension support such scenarios?

**Comment by badrange** [comment]
There is a PR that may help with this: #3972

**Comment by tobias-tengler** [comment]
Yes, I know about `pathToBinary`. What I'm talking about is, if the package is installed via PnP you only have a zipped package file in your repository. Without some additional changes I don't think the `pathToBinary` would be able to run the executable in the archive. I haven't tried it though, so I'll check if it works.
